app/frontend/src/components/base/BaseCopyToClipboard.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <span>
+    <v-btn
+      color="blue"
+      text
+      small
+      v-clipboard:copy="copyText"
+      v-clipboard:success="clipboardSuccessHandler"
+      v-clipboard:error="clipboardErrorHandler"
+    >
+      <v-icon class="mr-1">share</v-icon>
```

Comment:
Should this be a copy icon, not a share?

Reply:
damn, yeah